src/lib/serviceRoutinesV2/optionsAllNotDelete.cpp

```diff
@@ -45,8 +46,11 @@ std::string optionsAllNotDelete
   ParseData*                 parseDataP
 )
 {
-  ciP->httpHeader.push_back("Access-Control-Allow-Methods");
-  ciP->httpHeaderValue.push_back("GET, PATCH, POST, PUT, OPTIONS");
+  if ( (ciP->httpHeaders.origin != "") && ((strcmp(corsOrigin, "__ALL") == 0) || (strcmp(ciP->httpHeaders.origin.c_str(), corsOrigin) == 0)) )
+  {
+    ciP->httpHeader.push_back("Access-Control-Allow-Methods");

```

Comment:
Should use ` ACCESS_CONTROL_ALLOW_METHODS`.

(Similar cases along others serviceRoutinesV2/ files in this PR)

Reply:
Fixed in 3e6e50a